CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.8.3 (2020-02-26)
+
+- FIxed the repeater component to not submit forms when the "Top" button on reorderable repeaters is clicked. [#128](https://github.com/blackbaud/skyux-lists/pull/128)
```

Comment:
Fixed the repeater component to prevent form submission when users click the “Top” button on reorderable repeaters.

?

Reply:
Sounds good. Change made